main/src/ca/uwaterloo/flix/api/lsp/provider/completion/ExprCompleter.scala

```diff
@@ -23,7 +23,6 @@ import ca.uwaterloo.flix.language.ast.TypedAst
 object ExprCompleter {
 
   def getCompletions(context: CompletionContext)(implicit flix: Flix, index: Index, root: TypedAst.Root): Iterable[Completion] = {
-    DefCompleter.getCompletions(context) ++
```

Comment:
I am still confused why you remove this.

If you remove it then I can no longer type `List.m` and have it completed to `List.map`?